src/components/dialog/dialog.spec.js

```diff
@@ -785,6 +784,81 @@ describe('$mdDialog', function() {
       }));
     });
 
+    describe('before close', function () {
+      it('should ask before close promise resolved should close modal', inject(function ($mdDialog, $mdConstant, $q) {
```

Comment:
couldn't understand what `should ask before close promise resolved should close modal` means?